public/themes/default.css

```diff
@@ -352,7 +351,6 @@ video.post-attachment {
   background-color: var(--colors-background);
   max-width: 100%;
   max-height: 384px;
-  object-fit: contain;
```

Comment:
This just controls the visual appearance of the cropping, & personally I prefer it so you can see the whole image (unlike Twitter or something). This shouldn’t change the layout

Reply:
oops ya I forgot to add that back